selectors/rhs.jsx

```diff
@@ -41,3 +43,43 @@ export const getSelectedPost = createSelector(
         };
     }
 );
+
+export function makeGetCurrentUsersLatestPost(channelId, rootId) {
+    return createSelector(
+        getCurrentUserId,
+        (state) => state.entities.posts.postsInChannel[channelId] || [],
+        (state) => (id) => getPost(state, id),
+        (userId, postIds, getPostById) => {
+            let lastPost = null;
+
+            for (const id of postIds) {
+                const post = getPostById(id) || {};
+
+                // don't edit webhook posts, deleted posts, or system messages
+                if (post.user_id !== userId ||
+                    (post.props && post.props.from_webhook) ||
+                    post.state === Constants.POST_DELETED ||
+                    (post.type && post.type.startsWith(Constants.SYSTEM_MESSAGE_PREFIX))) {
+                    continue;
+                }
+
+                if (rootId) {
+                    if (post.root_id === rootId || post.id === rootId) {
+                        lastPost = post;
+                        break;
+                    }
+                } else {
+                    lastPost = post;
+                    break;
+                }
+            }
+
+            return lastPost;
+        }
+    );
+}
+
+export function makeGetCommentDraft(rootId) {
```

Comment:
This looks a bit weird to me since it's not returning a selector here as I'd expect from a `makeGetX` function, but I assume that's because this is based off the storage helper functions. It's fine to keep as is though.